releasenotes/notes/passmanager_replace-d89e2cc46517d917.yaml

```diff
@@ -0,0 +1,35 @@
+---
+features:
+  - |
+    A given pass manager now can be edited with the new method `replace`. This method allows to
+    replace a particular stage in a pass manager, which can be handy when dealing with preset
+    pass managers. For example, let's edit the layout selector of the pass manager used at
+    optimization level 0:
+
+    ```python
```

Comment:
The release notes use ReStructured Text (just like the docs), not markdown for formatting. This will not rendere properly. You can refer to: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html for the syntax used.

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/3004/commits/0a7dd56507c339ed6e716c47910cd67aa7af7de6